tests/examples/common.py

```diff
@@ -0,0 +1,63 @@
+import os
+import tempfile
+import subprocess
+
+MOCOCRW_EXAMPLE_BINARY_PATH = "."
+try:
+    MOCOCRW_EXAMPLE_BINARY_PATH = os.environ["MOCOCRW_EXAMPLE_BINARY_PATH"]
+except KeyError:
+    print("Please set the MOCOCRW_EXAMPLE_BINARY_PATH environment variable.")
+    exit(1)
+
+MOCOCRW_DIR = ""
+try:
+    MOCOCRW_SRC_DIR = os.environ["MOCOCRW_SRC_DIR"]
+except KeyError:
+    print("Please set the MOCOCRW_SRC_DIR environment variable")
+    exit(1)
+
+
+HASH_ALGOS = [
+    "SHA256", "SHA384", "SHA512", "SHA3-256", "SHA3-384", "SHA3-512"
+]
+
+
+def add_invalid_parameters_and_execute(execute_string):
+    def execute(exec_string):
+        result = subprocess.run(exec_string.split(), capture_output=True)
+        assert result.returncode == 1
+        assert b"Unknown option selected: unrecognised option '--aw\xc3\xb6eljf'\n" \
+               b"Failure parsing command line.\n" == result.stderr
+    execute(execute_string + " --awöeljf asdölvkjyxc")
+
+
+def create_temp_working_dir():
+    # Create working directory
+    new_working_dir = tempfile.mkdtemp()
+
+    current_working_dir = os.getcwd()
+    # Change to working directory
+    os.chdir(new_working_dir)
+    return current_working_dir, new_working_dir
+
+
+def combine_all_options(all_options, option_list):
+    first_options_list = option_list[0]
+    add_options = [add_opt + opt for add_opt in first_options_list for opt in all_options]
+    all_options += add_options
+    if len(option_list) == 1:
+        return all_options
+    return combine_all_options(all_options, option_list[1:])
+
+
+def build_option_list(options_dict):
+    if len(options_dict) == 0:
```

Comment:
Usually, that's written as
```suggestion
    if not options_dict:
```
in Python.